projects/angular/src/emphasis/alert/alerts-pager.spec.ts

```diff
@@ -99,6 +100,55 @@ export default function () {
         expect(this.fixture.componentInstance.currentAlert).toEqual(this.alert);
       });
     });
+
+    describe('View Basics', function () {
+      let fixture: ComponentFixture<TestBasics>;
+      let pagerInstance: ClrAlertsPager;
+      let commonStrings: ClrCommonStringsService;
+      const testNextAriaLabel = (current: number, count: number) => {
```

Comment:
A couple things here:

1. This function does test anything, so I would rename to `get...` instead of `test...`.
2. I think this should be a function declaration instead of a variable declaration with a function value. I think a first-class function declared after the specs (out of the way) would make the tests a bit easier to read.